src/bin/rdcore/cmdline.rs

```diff
@@ -32,11 +37,30 @@ pub fn parse_args() -> Result<Config> {
         .global_setting(AppSettings::DisableHelpSubcommand)
         .global_setting(AppSettings::UnifiedHelpMessage)
         .global_setting(AppSettings::VersionlessSubcommands)
+        .subcommand(
+            SubCommand::with_name("stream-hash")
+                .about("Copy data from stdin to stdout, checking piecewise hashes")
+                .arg(
+                    Arg::with_name("hash-file")
+                        .value_name("hash-file")
```

Comment:
We've been a bit inconsistent about this, though WDYT about using capital letters as a convention for value names? So e.g. `HASH_FILE`.

Obviously not a blocker.

Reply:
Grepping for `value_name`, I'd say I've been consistent and you've been consistent.  :grin:  Personally I find the upper-case less readable.